index.html

```diff
@@ -0,0 +1,754 @@
+    <link rel="stylesheet" href="assets/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+
+    <script src="https://kit.fontawesome.com/f58a22ad21.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+    <header>
+        <nav>
+            <div class="justFlex">
+                <div class="logo3">
+                    <img src="assets/images/logo3.PNG" alt="logo3">
+                </div>
+                <div class="screenMenu">
+                    <div class="menu flex-spaceBetween">
+                        <ul class="menu-item">
+                            <li><a href="#">News</a></li>
+                            <li><a href="#">Events</a></li>
+                            <li><a href="#">Business</a></li>
+                            <li><a href="#">AMAs</a></li>
+                            <li><a href="#">Spaces</a></li>
+                            <li><a href="#">Terms & Conditions</a></li>
+                        </ul>
+                        <ul class="menu-item">
+                            <li><a href="#">About</a></li>
+                            <li><a href="#">Advertise</a></li>
+                            <li><a href="#">Jobs</a></li>
+                            <li><a href="#">Contact</a></li>
+                        </ul>
+                    </div>
+                    <div class="menu2 flex-spaceBetween">
+                        <ul class="menu-item2">
+                            <li><a href="#">LATEST</a></li>
+                            <li><a href="#">HARD FORK</a></li>
+                            <li><a href="#">PLUGGED</a></li>
+                            <li><a href="#">README</a></li>
+                            <li><a href="#">GROWTH QUARTERS</a></li>
+                            <li><a href="#">NEURAL</a></li>
+                        </ul>
+                        <ul>
+                            <li><i class="fas fa-search"></i></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>    
+
+            <div class="menuMobile">
+                <div class="mobileMenu flex-spaceBetween">
+                    <div><img src="assets/images/logo2.PNG" alt="logo-image"></div>
+                    <div>
+                        <i class="fas fa-search"></i>
+                        <i class="fas fa-bars"></i>
+                    </div>
+                </div>
+                <div class="weHeard">
+                    We heard you like events and sweatpants so we made event you
+                    can join in your seatpants <i class="fas fa-long-arrow-alt-right"></i>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <section class="news-section flex">
+        <div class="news__article1">
+            <div class="feature-main__top">
+                <img src="assets/images/tnw.svg" alt="" class="feature-article__logo" />
```

Comment:
The TNW logo in desktop mode is sticking to the corner and smaller than the original. The logo should be bigger and must not stick to the corner.
Right now 👇
![Annotation 2020-06-04 230308](https://user-images.githubusercontent.com/57726348/83791751-c99da800-a6b7-11ea-9eb8-d8b8c7a774ad.jpg)

It should be 👇
![Annotation 2020-06-04 230247](https://user-images.githubusercontent.com/57726348/83791799-d7ebc400-a6b7-11ea-988f-18c98ed16520.jpg)